examples/README.md

```diff
@@ -38,6 +38,19 @@ This simple app shows how to use an encrypted realm.
 
 This simple app demonstrates how to define models with inverse relationships using `-linkingObjectsOfClass:forProperty:`.
 
+### Draw
+
+This is a simple drawing app designed to show off the collaborative features of the [Realm Mobile Platform](https://realm.io/news/introducing-realm-mobile-platform/).
+
+Any number of users may draw on a single shared canvas in any given moment, with contributions from other devices appearing on the canvas in real-time.
+
+#### Installation Instructions
+
+1. [Download the macOS version](https://realm.io/docs/realm-mobile-platform/get-started/) of the Realm Mobile Platform.
+2. Run a local instance of the Realm Mobile Platform.
+3. Create a user with the email 'demo@realm.io' and the password 'demo'.
```

Comment:
could you specify where/how this user is created? should be in the ROS dashboard, right?